src/code/z_parameter.c

```diff
@@ -4366,527 +4364,489 @@ void Interface_DrawClock(PlayState* play) {
-                    gSP1Quadrangle(OVERLAY_DISP++, 4, 6, 7, 5, 0);
-
-                    /**
-                     * Draws Three-Day Clock's Hour Digit Above the Moon
-                     */
-                    // Rotates Three-Day Clock's Hour Digit To Above the Moon
-                    Matrix_Translate(0.0f, R_THREE_DAY_CLOCK_Y_POS / 10.0f, 0.0f, MTXMODE_NEW);
-                    Matrix_Scale(1.0f, 1.0f, 1.0f, MTXMODE_APPLY);
-                    Matrix_RotateZF(-sp1CC, MTXMODE_APPLY);
-                    gSPMatrix(OVERLAY_DISP++, Matrix_NewMtx(play->state.gfxCtx),
-                              G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            // Rotates Three-Day Clock's Hour Digit To Above the Sun
+            Matrix_Translate(0.0f, R_THREE_DAY_CLOCK_Y_POS / 10.0f, 0.0f, MTXMODE_NEW);
+            Matrix_Scale(1.0f, 1.0f, 1.0f, MTXMODE_APPLY);
+            Matrix_RotateZF(-(sp1CC - 3.15f), MTXMODE_APPLY);
+            gSPMatrix(OVERLAY_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
 
-                    // Draws Three-Day Clock's Hour Digit Above the Moon
-                    gDPPipeSync(OVERLAY_DISP++);
-                    gDPSetCombineLERP(OVERLAY_DISP++, 0, 0, 0, PRIMITIVE, TEXEL0, 0, PRIMITIVE, 0, 0, 0, 0, PRIMITIVE,
-                                      TEXEL0, 0, PRIMITIVE, 0);
-                    gDPSetPrimColor(OVERLAY_DISP++, 0, 0, 0, 0, 0, sThreeDayClockAlpha);
-                    gSPVertex(OVERLAY_DISP++, &interfaceCtx->actionVtx[32], 8, 0);
+            // Draws Three-Day Clock's Hour Digit Above the Sun
+            gDPPipeSync(OVERLAY_DISP++);
+            gDPSetCombineLERP(OVERLAY_DISP++, 0, 0, 0, PRIMITIVE, TEXEL0, 0, PRIMITIVE, 0, 0, 0, 0, PRIMITIVE, TEXEL0,
+                              0, PRIMITIVE, 0);
+            gDPSetPrimColor(OVERLAY_DISP++, 0, 0, 0, 0, 0, sThreeDayClockAlpha);
+            gSPVertex(OVERLAY_DISP++, &interfaceCtx->actionVtx[24], 8, 0);
 
-                    OVERLAY_DISP = Gfx_DrawTexQuad4b(OVERLAY_DISP, sThreeDayClockHourTextures[sp1C6], 4, 16, 11, 0);
+            OVERLAY_DISP = Gfx_DrawTexQuad4b(OVERLAY_DISP, sThreeDayClockHourTextures[sp1C6], 4, 16, 11, 0);
 
-                    // Colours the Three-Day Clocks's Hour Digit Above the Moon
-                    gDPPipeSync(OVERLAY_DISP++);
-                    gDPSetPrimColor(OVERLAY_DISP++, 0, 0, 255, 255, 155, sThreeDayClockAlpha);
-                    gSP1Quadrangle(OVERLAY_DISP++, 4, 6, 7, 5, 0);
+            // Colours the Three-Day Clocks's Hour Digit Above the Sun
+            gDPPipeSync(OVERLAY_DISP++);
+            gDPSetPrimColor(OVERLAY_DISP++, 0, 0, 255, 255, 155, sThreeDayClockAlpha);
+            gSP1Quadrangle(OVERLAY_DISP++, 4, 6, 7, 5, 0);
+
+            /**
+             * Draws Three-Day Clock's Hour Digit Above the Moon
+             */
+            // Rotates Three-Day Clock's Hour Digit To Above the Moon
```

Comment:
Nit: Is there a specific reason to have the different style of commenting? could these be combined?

Reply:
The idea I was trying to go for was that this function is incredibly long (500+ lines in C), so the /* comments are for describing entire blocks of code (like sections or chapter titles), while the // comments are for describing 1-2 line comments. I think this is the only place where there's both which is why it's confusing. I put an extra line gap between them, but other suggestions are welcome

Reply:
I might add a `section:`/`block:` for the /* comments to help clarify.

Reply:
I put in `Section: ` for them